Add Status Bar configuration via Cordova Plugin

ssoriche opened the pull request:
Configure the Status Bar with plugin options hardcoded. Phonegap will
do the right thing on its own with the values for mobile devices, and
does not involve Javascript or CSS "trickery".

The issue with this change is the background colour is set to black
with no way to make it transparent, or match the background of the
application.

Note, this change may require a new IPA as Phonegap might need it to
handle the config.xml changes.

dasilva333 commented:
So does this change hide the status bar or show the status bar?  I'm not sure that's why I'm asking. I reverted the previous status bar changes because the css transform was messing with the transfer popup's position:fixed style.

I'm thinking of a middle ground solution where the statusbar can always show when you're scrolled all the way to the top and then hides as you scroll or after you scroll past a certain set of pixels, so it's like part of the page. 

Let me know thanks

ssoriche commented:
This is an always show. Wanted to get it in and fixed so that the menu doesn’t overlap first.

I had been working on making it sticky using position: fixed style too, but as you say, started making a mess.

Next I can look at making the status bar hide itself when scrolled down screen size + 20px, but this fix at least makes it appear, and doesn’t mess anything else up.

~Shawn


> On Apr 11, 2015, at 12:52 PM, Richard Pinedo <notifications@github.com> wrote:
> 
> So does this change hide the status bar or show the status bar? I'm not sure that's why I'm asking. I reverted the previous status bar changes because the css transform was messing with the transfer popup's position:fixed style.
> 
> I'm thinking of a middle ground solution where the statusbar can always show when you're scrolled all the way to the top and then hides as you scroll or after you scroll past a certain set of pixels, so it's like part of the page.
> 
> Let me know thanks
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dasilva333/TowerGhostForDestiny/pull/19#issuecomment-91880137>.
>
